recipe/0001-Relax-version-constraint-on-importlib-metadata.patch

```diff
@@ -1,25 +1,25 @@
-From 8e5e17bbe9b51546c9e21c483c0c66cb5d2560ee Mon Sep 17 00:00:00 2001
-From: Chris Burr <christopher.burr@cern.ch>
-Date: Sat, 26 Jun 2021 10:39:38 +0200
+From a1ac19d89e2e6a17fda87b860d4ed3a6adf363e6 Mon Sep 17 00:00:00 2001
+From: AJ Schmidt <aschmidt@nvidia.com>
+Date: Tue, 17 Aug 2021 10:27:35 -0400
 Subject: [PATCH] Relax version constraint on importlib-metadata
 
 ---
- pyppeteer-0.2.5/pyproject.toml | 2 +-
+ pyppeteer-0.2.6/pyproject.toml | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
-diff --git a/pyppeteer-0.2.5/pyproject.toml b/pyppeteer-0.2.5/pyproject.toml
-index 0dbe8b5..3453e54 100644
---- a/pyppeteer-0.2.5/pyproject.toml
-+++ b/pyppeteer-0.2.5/pyproject.toml
-@@ -47,7 +47,7 @@ pyppeteer-install = 'pyppeteer.command:install'
- python = "^3.6.1"
- appdirs = "^1.4.3"
- # todo: update this dep when tox releases patch version w/ https://github.com/tox-dev/tox/pull/1764
--importlib-metadata = { version = "^2.1.1", python = "<3.8" }
-+importlib-metadata = { version = ">=2.1.1", python = "<3.8" }
- pyee = "^8.1.0"
- tqdm = "^4.42.1"
- urllib3 = "^1.25.8"
--- 
-2.28.0
+diff --git a/pyppeteer-0.2.6/pyproject.toml b/pyppeteer-0.2.6/pyproject.toml
+index 31bb753..d3045a7 100644
+--- a/pyppeteer-0.2.6/pyproject.toml
++++ b/pyppeteer-0.2.6/pyproject.toml
+@@ -46,7 +46,7 @@ pyppeteer-install = 'pyppeteer.command:install'
+ [tool.poetry.dependencies]
+ python = "^3.6.1"
+ appdirs = "^1.4.3"
+-importlib-metadata = ">=1.4"
++importlib-metadata = { version = ">=2.1.1", python = "<3.8" }
```

Comment:
This has also been loosened, and we likely no longer need the patch:
https://github.com/pyppeteer/pyppeteer/blame/0.2.6/pyproject.toml#L49